lib/arrays.md

```diff
@@ -5,44 +5,52 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is composed of values stored in cells. It is defined by brackets in Ruby.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+An array and a string can form the same block of text, but because an array can
+store words of the text into its cells. Strings are composed of single values.
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+If an array has n elements then to call the nth element the command array_name[n].
+When a n+1 element is asked then a nil response will be given.
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
 
-Replace me with your response...
+Just like with strings, asking for a negative value for a element will start the
+call from the end of the list of elements.
 
 # How would you perform an operation on every element inside an Array?
 
-Replace me with your response...
+array_name*2 will multiply each element by 2. 
```

Comment:
I assumed we were talking about numbers when the question used "multiply". To multiply each element of given characteristic, I would use the arr.each{|x| arr[x]*2}